MdePkg/Test/Mock/Library/GoogleTest/Ppi/MockPeiReportStatusCodeHandler.cpp

```diff
@@ -0,0 +1,21 @@
+/** @file MockPeiReportStatusCodeHandler.cpp

```

Comment:
Can you delete this cpp file and move the definitions to the header file? This is a new guideline for protocols/ppis since they don't have infs.

Same for the rest of them.

Reply:
Updated the CPP changes in Header files and removed CPP files.